tactics/tactics.ml

```diff
@@ -1823,7 +1823,9 @@ let general_apply ?(respect_opaque=false) with_delta with_destruct with_evars
           else
             Proofview.tclZERO ~info exn0 in
         if not (Int.equal concl_nprod 0) then
-          Tacticals.New.tclORELSE0 (try_apply thm_ty 0) tac
+          Proofview.tclORELSE
+            (try_apply thm_ty 0)
+            (fun _ -> tac)
```

Comment:
This one does not change anything because `tac` is already a value.

Reply:
Yes. You would prefer keeping a `Tacticals.New.tclORELSE` instead? I started from a full revert, this is why it went back to `Proofview.tclORELSE`.

Reply:
I'd prefer indeed, this is vaguely confusing to thunk for nothing.